spec/lib/utils/job-utils/racadm-parser-spec.js

```diff
@@ -0,0 +1,151 @@
+                currentVersion: 'TS04',
+                elementName: 'Disk 2 in Backplane 0 of RAID Controller in Slot 1',
+                availableVersion: '',
+                rollbackVersion: ''
+            });
+        });
+    });
+
+/*
+    describe("XML to Json Parser", function() {
+        it("should convert XML file to Json format", function() {
+            var data = racadmOutMock.racadmXml;
+            console.log(typeof data);
+            var result = parser.xmlToJson(data);
+
+        });
+    });
+*/
+
+    describe("Parse file path", function() {
+        it("should parse  remote filename and path", function() {
+            var data = "//192.168.191.207/share/bios.xml";
+            var result = parser.getPathFilename(data);
+            expect(result.path).to.equal('//192.168.191.207/share');
+            expect(result.name).to.equal('bios.xml');
+            expect(result.style).to.equal('remote');
+        });
+
+        it("should parse  remote filename and path", function() {
+            var data = "/home/share/bios.xml";
+            var result = parser.getPathFilename(data);
+            expect(result.path).to.equal('/home/share');
+            expect(result.name).to.equal('bios.xml');
+            expect(result.style).to.equal('local');
+        });
+
+        it("should report path format incorrect", function() {
+            var data = "home/share/bios.xml";
+            expect(function() {
+                parser.getPathFilename(data);
+            }).to.throw(Error);
+        });
+
+    });
+
+    describe("Parse  JID", function() {
+        it("should filter JID from console standard output", function() {
+            var data = racadmOutMock.racadmJobIdData;
+            var result = parser.getJobId(data);
+            expect(result).to.equal('JID_541335487816');
+        });
+
+        it("should report can't parser JID", function() {
+            var data = 'any string';
+            expect(function() {
+                parser.getPathFilename(data);
+            }).to.throw(Error);
+        });
+
+    });
+
+    describe("Parser job status", function() {
+        it("should parser job status", function() {
+            var data = racadmOutMock.racadmJobStatusData;
+            var result = parser.getJobStatus(data);
+            expect(result.jobId).to.equal('JID_541347990377');
+            expect(result.jobName).to.equal('Configure: Import system configuration XML file');
+            expect(result.status).to.equal('Completed');
+            expect(result.startTime).to.equal('Not Applicable');
+            expect(result.expirationTime).to.equal('Not Applicable');
+            expect(result.message).to.equal('SYS054: No configuration changes requiring a system ' +
+                'restart need to be applied.');
+            expect(result.percentComplete).to.equal('100');
+        });
+
+        it("should report job status format is not correct", function() {
```

Comment:
Minor comment here: change the description to "should throw error when report job status format is not correct" would be easier to understand.